pom.xml

```diff
@@ -461,6 +517,11 @@
       </dependency>
 
       <!-- Jackson -->
+      <dependency>
+        <groupId>com.fasterxml.jackson.module</groupId>
+        <artifactId>jackson-module-jakarta-xmlbind-annotations</artifactId>
+        <version>2.13.2</version>
```

Comment:
Minor: Please move version to properties section

Reply:
fixed in d368194853e5d80c261c1881cd29d45a05c56d2d